draft-ietf-tls-esni.md

```diff
@@ -1718,7 +1720,7 @@ response. In particular, the compression mechanism described in
 ClientHelloOuter to construct ClientHelloInner, or a buggy server may
 incorrectly apply parameters from ClientHelloOuter to the handshake.
 
-To begin, the attacker first interacts with a server to obtain a resumption
+To begin, the attacker starts interacting with a server to obtain a resumption
```

Comment:
The original text is better grammaticaly.

Reply:
Ok a few things here ... in French when you start with 'first' or 'firstly' ... you expect a 'second' or 'secondly' ... 

In several cases I left the 'first' because it was followed by a 'then' which is less coherent but in several places you have 'first' and not even a 'then'

Then I don't remember but I believe when I learned English that 'start' and 'stop' will mean 'ing' at the end of the next verb ...

Reply:
It would be better to just remove "first" then, so it would read: "To begin, the attacker interacts with..."

Reply:
Good for me actually!